examples/piggybank/src/Version1.tsx

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-console */
```

Comment:
I think there could be better reuse between Version0 and Version1.

Reply:
But we can defer that to later, as I believe there are more important things to work on right now.